File: domain/src/main/java/com/ortin/gesturetranslator/domain/repository/RecognizeCoordinateRepository.java
package com.ortin.gesturetranslator.domain.repository;

import androidx.annotation.NonNull;
import com.ortin.gesturetranslator.domain.models.CoordinateClassification;
import com.ortin.gesturetranslator.domain.models.ImageDetected;

public interface RecognizeCoordinateRepository {
    CoordinateClassification recognise(@NonNull ImageDetected imageDetected);
}

File: domain/src/main/kotlin/com/ortin/gesturetranslator/domain/repository/RecognizeCoordinateRepository.kt
package com.ortin.gesturetranslator.domain.repository

import com.ortin.gesturetranslator.domain.models.CoordinateClassification
import com.ortin.gesturetranslator.domain.models.ImageDetected

interface RecognizeCoordinateRepository {
    fun recognise(imageDetected: ImageDetected): CoordinateClassification
}
